libraries/grpc-sdk/src/classes/GrpcServer.ts

```diff
@@ -12,9 +12,11 @@ export class GrpcServer {
     protoDescription: string;
     functions: { [name: string]: Function };
   }[] = [];
+  private readonly grpcKey?: string;
 
-  constructor(port?: string) {
+  constructor(port?: string, grpcKey?: string) {
     this._url = `0.0.0.0:${ port ?? '5000' }`;
+    this.grpcKey = grpcKey;
```

Comment:
should be retrieved from env directly, there's no reason for this to be an argument in the constructor